.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2.0
+
+jobs:
+        build:
+                docker:
+                        - image: circleci/golang:1.12
+                working_directory: /go/src/github.com/pokt-network/pocket-core-deployments
+                steps:
+                        - checkout
+                        - setup_remote_docker
+                #Build the image                         
+                        - run: sudo docker build --build-arg BRANCH_NAME=staging -t test:test -f docker/Dockerfile docker/.
```

Comment:
@nelcolon 
Instead of definning a static variable BRANCH_NAME. you can use circle ci env variables for this. Just use $BRANCH_NAME 
https://circleci.com/docs/2.0/env-vars/

Reply:
Perfect!, just added that change on my latest commit.